### DISCUSSION TITLE
DBeaver - JDBC Driver Setup

### DISCUSSION THREAD

**Discussion by gv8349** [dbeaver/dbeaver#11504] [Q&A] [answered]
#### System information: 
- Windows 10
- DBeaver version - Version 7.3.5
- Java version -- Not Sure
- Additional extensions

#### Connection specification:
- Database name and version
- Driver name
- Do you use tunnels or proxies (SSH, SOCKS, etc)? 

#### Describe the problem you're observing:
I'm not able to install the jdbc drivers that are needed and not able to download and isntall JDBC drivers for teradata.

**Comment by kseniiaguzeeva** [comment, marked as answer]
You can do it. Here is the instruction https://github.com/dbeaver/dbeaver/wiki/Database-drivers. Ask to reopen the ticket, if you need some other help.
